A32NX/html_ui/Pages/A32NX_Core/A32NX_TransitionAltitude.js

```diff
@@ -0,0 +1,345 @@
+class A32NX_TransitionAltitude {
+    init() {
+        console.log('A32NX_TransitionAltitude init');
+        this.currentDeparture = "";
+        this.currentArrival = "";
+        this.offline = false;
+        this.checkStart = true;
+        this.checkStartSec = 0;
+        this.testAirport = "";
+        this.trySendRequest();
+    }
+
+    update(_deltaTime, _core) {
+        if (this.offline === true) {
+            this.offlineTransAlt();
+        }
+        this.tryCheckAPI(_deltaTime);
```

Comment:
**question**: should this function be called when `this.offline` is true?

Reply:
YES. Only use offline db when our api server is down

Reply:
So when you have figured out you are in `offline`, you still need to call `this.tryCheckAPI(_deltaTime);` to do what exactly?

Reply:
that removed on current code. this is outdated